### PR TITLE
[strings] Updated taxi stand translations

### PR DESCRIPTION
**Pull request by map-per**
Updated taxi stand translations as suggested in https://github.com/organicmaps/organicmaps/pull/5333/commits/9690a7a94b2bb2741e9b14c342182c8759268145#r1234762573.

> @vng will taxi be searchable by a single taxi word in all these cases above?

Typing of taxi should suggest 'taxi stand' in all languages.

### CODE REVIEW

**Comment by map-per** [comment]
So you would use e.g. "Taxi Stand" in types_strings.txt and "Taxi" for the search cathegory?

**Comment by vng** [comment]
I'd prefer "Taxi" only everywhere. But let's listen to other folks.

File: data/categories.txt [hunk 1]
```diff
@@ -2632,41 +2632,45 @@ sw:Bandali|kivuko
...
-ru:3Такси
-bg:3такси|шофьор
-ar:سيارة أجرة|تاكسي|مواصلات|مواصلات
-cs:3Taxi
-da:3Taxi|taxa
-nl:3Taxi
-fi:3Taksi
-fr:3Taxi
-de:3Taxi|Taxistand|Taxe|ÖPNV
+en:3Taxi Stand|Taxi Rank|Taxi|U+1F695|U+1F696
```
Comment (biodranik) [inline data/categories.txt:2635]: I would put 3 to taxi for language neutrality.

File: data/categories.txt [hunk 2]
```diff
@@ -2632,41 +2632,45 @@ sw:Bandali|kivuko
...
+ar:سيارة أجرة|تاكسي|مواصلات|مواصلات|محطة التاكسي
+be:Стаянка таксі
```
Comment (biodranik) [inline data/categories.txt:2637]: The single word "taxi" should stay as a synonym everywhere, in addition to localized single word "taxi" and localized "taxi stand" translation.

Comment (map-per) [reply data/categories.txt:2637]: Is it really necessary to explicitly add "taxi" to all languages? If `en:3Taxi` is present searching for "taxi" should work in all languages or?

Comment (biodranik) [reply data/categories.txt:2637]: @vng can you please confirm that en synonyms are always used by the search?

Comment (vng) [reply data/categories.txt:2637]: Confirm

File: data/categories.txt (hunk 1 above)
Comment (biodranik) [reply data/categories.txt:2635]: @vng Is there a way to see Taxi Rank suggestion in the US, and Taxi Stand in the UK? If there is no one, I would suggest showing just "Taxi" hint for language neutrality. Maybe not everyone in the US knows what is Taxi Stand.

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -4061,31 +4061,44 @@
...
+    pl = Postój taksówek
+    pt = Ponto de táxi
+    ro = Stație de taxi
+    ru = Стоянка такси
+    sk = Stanovište taxíkov
+    sv = Taxistation
+    sw = Stendi ya teksi
+    th = ป้ายรอแท็กซี่
+    tr = Taksi durağı
+    uk = Стоянка таксі
```
Comment (biodranik) [inline data/strings/types_strings.txt:4098]: Зупинка таксі  ? @strump

**Comment by vng** [comment]
At this moment, the best option here is to remove the stuff like "stand, rank, стоянка, зупинка"!

File: data/strings/types_strings.txt (hunk 3 above)
Comment (strump) [reply data/strings/types_strings.txt:4098]: `Стоянка таксі` - is correct. It means where taxi starts. While `Зупинка таксі` means place where taxi finishes.

**Comment by map-per** [comment]
Closing this PR, the current translations are good, no need for change.

**Pull request closed without merging**